src/app/sevices/auth.service.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpClient } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { authDetails } from "../models/authDetails";
+import { authStatus } from "../models/authStatus";
+import { backend_auth } from "../constants/endpoints";
+import { Observable, tap } from "rxjs";
+import { UserService } from "./user.service";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class AuthService {
+
+    constructor(private http: HttpClient, private userService: UserService) {};
+
+    // POST /login
+    // returns a JWT to store in cookies
+    login(authDetails: authDetails): Observable<authStatus> {
```

Comment:
authStatus uppercase also I noticed that models are named authStatus and authDetails etc, name them in kebab-case so
auth-status.model.ts and auth-details.model.ts